MediaBrowser.Controller/Entities/Folder.cs

```
@@ -377,7 +377,7 @@ private async Task ValidateChildrenInternal2(IProgress<double> progress, Cancell
                 {
                     nonCachedChildren = GetNonCachedChildren(directoryService);
                 }
-                catch (Exception ex)
+                catch (Exception)
```

Why not log all of these errors rather than silently ignore them?